src/pages/workshops/WorkshopAttendance.js

```diff
@@ -0,0 +1,124 @@
+import React, { useEffect, useState } from "react";
+import { json, Link, useLocation } from "react-router-dom";
+import Select from "react-select";
+import "./Workshops.css";
+
+export const WorkshopAttendance = () => {
+    const workshopID = useLocation().state.id;
+    console.log(workshopID);
+    const [workshop, setWorkshop] = useState({});
+    const [idMode, setidMode] = useState(false);
+    const [id, setID] = useState(0);
+    const [totalAttendees, setTotalAttendees] = useState(0);
+    const [registered, setRegistered] = useState(0);
+    const [ratingPoints, setRatingPoints] = useState(0);
+    const [benIDs, setBenIds] = useState([]);
+    const [message, setMessage] = useState("");
+    const handleRating = (rating) => {
+        setRatingPoints(ratingPoints + rating);
+        setTotalAttendees(totalAttendees + 1);
+        setidMode(true);
+    };
+    const handleIDNumber = (event) => {
```

Comment:
maybe rename this function to be more clear - is this more like `handleIdNumberChange`